sdk/iOS/src/MSTable.h

```diff
@@ -162,12 +162,12 @@ extern NSString *const MSSystemColumnDeleted;
 /// configured and then executed to retrieve items from the table. An *MSQuery*
 /// instance provides more flexibilty when querying a table than the table
 /// read* methods.
--(MSQuery *)query;
+-(nonnull MSQuery *)query;
 
 /// Returns an *MSQuery* instance associated with the table that uses
 /// the given predicate. An *MSQuery* instance provides more flexibilty when
 /// querying a table than the table read* methods.
--(MSQuery *)queryWithPredicate:(NSPredicate *)predicate;
+-(nonnull MSQuery *)queryWithPredicate:(nullable NSPredicate *)predicate;
```

Comment:
seems like this one could be nonnull, why use it if you aren't passing a predicate.